runk/Dockerfile

```diff
@@ -3,6 +3,7 @@
 FROM golang:1.12-alpine AS sbox
```

Comment:
why did you move this under runk if it builds sbox.

Reply:
My thinking was that it never hurts to have a Dockerfile for Go dev or test env, but I can put it in sbox, but not sure about README (see below).